### PR TITLE
Fix metadata_store_service_go_proto build error

### PR DESCRIPTION
**Pull request by guunergooner**
* bugfix
[Build metadata store go proto and metadata store service go proto error](https://github.com/google/ml-metadata/issues/112)

@hughmiao Help review PR

### CODE REVIEW

**Comment by tongchao199** [comment]
@googlebot I signed it!

File: WORKSPACE [hunk 1]
```diff
@@ -52,21 +52,21 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
     name = "io_bazel_rules_go",
     urls = [
-        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/v0.20.3/rules_go-v0.20.3.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.20.3/rules_go-v0.20.3.tar.gz",
+        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/0.17.0/rules_go-0.17.0.tar.gz",
```
Comment (hughmiao) [inline WORKSPACE:55]: Lowering the `io_bazel_rules_go` and `bazel_gazelle` versions had issues to build other targets in the workspace (e.g., pip wheels, and cc). let's keep them as is for now?  Please also see the background in the metadata_store/BUILD comment below.

File: ml_metadata/metadata_store/BUILD [hunk 2]
```diff
@@ -647,3 +647,40 @@ cc_test(
         "@org_tensorflow//tensorflow/core:test",
     ],
 )
+
+ml_metadata_go_wrap_cc(
```
Comment (hughmiao) [inline ml_metadata/metadata_store/BUILD:651]: qq, will you only depend on the proto targets? if so, shall we exclude these targets?

Some background, the go libs were excluded temporarily due to the go-rules had issues with py3.8 / py3.9 integration testing images. The grpc, proto dependencies currently were introduced by TF workspace. We had a todo to reintroduce them back once drop the TF's dependency in the WORKSPACE.

**Review by hughmiao — COMMENTED** [review]
@tongchao199 , thanks for the contribution!

**Pull request closed without merging**